src/bench/ccoins_caching.cpp

```diff
@@ -4,8 +4,8 @@
 
 #include <bench/bench.h>
 #include <coins.h>
+#include <script/signingprovider.h>
```

Comment:
Nit: `p` comes before `s` (move this one line down).

Reply:
Fixed the order here and everywhere.